### DISCUSSION TITLE
是否有编译、运行项目的合适方法？

### DISCUSSION THREAD

**Discussion by XZirui** [ayamir/nvimdots#1056] [General]
我正在使用nvimdots的配置，我对此感到非常满意，也非常感谢开发者的工作

当我尝试用neovim去开发项目时，我没在当前的配置中找到一个与编译、运行项目有关的插件
以C++为例，在vscode上有cmake等插件供我一键编译并运行项目，在vs和clion上都有自带的对应功能
我想知道在当前配置上，是有此功能，但我未找到；还是没有此功能，需要自行搭配其他插件？

**Comment by ayamir** [comment]
Yes, you should install and configure a plugin like this: https://github.com/Civitasv/cmake-tools.nvim.

**Comment by XZirui** [comment]
OK. The problem has solved.
